python_modules/libraries/dagster-dbt/dagster_dbt/asset_defs.py

```diff
@@ -471,16 +475,19 @@ def _dbt_nodes_to_assets(
 def load_assets_from_dbt_project(
     project_dir: str,
     profiles_dir: Optional[str] = None,
-    target_dir: Optional[str] = None,
+    *,
     select: Optional[str] = None,
     exclude: Optional[str] = None,
+    dagster_dbt_translator: Optional[DagsterDbtTranslator] = None,
+    io_manager_key: Optional[str] = None,
+    target_dir: Optional[str] = None,
```

Comment:
Would argue for deprecating this. This is only needed when compiling the initial manifest. This can be hardcoded as `target`.

Reply:
What do you mean by "hardcoded as target"?

Reply:
Thoughts @OwenKephart ?

Reply:
Meaning we can use the default target path, which is `target/` in the dbt project directory.